src/index.ts

```diff
@@ -17,36 +17,47 @@
  *
  */
 
+import * as logdown from 'logdown';
+
 import config from './config';
 import Server from './Server';
 import {formatDate} from './utils';
 
+const logger = logdown('@wireapp/wire-web-ets/index', {
+  logger: console,
+  markdown: false,
+});
+
 const server = new Server(config);
 
 server
   .start()
-  .then(port => console.info(`[${formatDate()}] Server is running on port ${port}.`))
-  .catch(error => console.error(`[${formatDate()}] ${error.stack}`));
+  .then(port => logger.info(`[${formatDate()}] Server is running on port ${port}.`))
```

Comment:
note for the future: we might want to have a wrapper on top of `logdown` that will take care of generating this `[${formatDate()]` for us ;)